plugins/org.jkiss.dbeaver.tasks.native.ui/src/org/jkiss/dbeaver/tasks/ui/nativetool/internal/TaskNativeUIMessages_zh.properties

```diff
@@ -41,19 +41,20 @@ tools_wizard_page_dialog_button_all = \u5168\u9009
 
 tools_wizard_page_dialog_button_none = \u5168\u4E0D\u9009
 
+tools_wizard_page_dialog_file_pattern = \u6587\u4EF6\u540D\u8868\u8FBE\u5F0F
+
+tools_wizard_page_dialog_folder_pattern = \u8F93\u51FA\u6587\u4EF6\u5939
+
 tools_wizard_page_dialog_label_extra_command_args = \u989D\u5916\u7684\u547D\u4EE4\u53C2\u6570
 
 tools_wizard_page_dialog_tooltip_extra_command_args = \u4E3A\u5DE5\u5177\u53EF\u6267\u884C\u6587\u4EF6\u8BBE\u7F6E\u989D\u5916\u7684\u547D\u4EE4\u53C2\u6570\u3002
 
-tools_wizard_page_log_task_finished = {0} \u5B8C\u6210\u4E8E {1}
+tools_wizard_page_log_task_finished = {0} \u7ED3\u675F\u4E8E {1}
 
 tools_wizard_page_log_task_log_reader = {0} \u8BFB\u53D6\u65E5\u5FD7
 
-tools_wizard_page_log_task_progress = {0} \u8FDB\u5EA6
+tools_wizard_page_log_task_progress = {0}\u8FDB\u5EA6
 
-tools_wizard_page_log_task_progress_log = {0} \u8FDB\u5EA6\u65E5\u5FD7
+tools_wizard_page_log_task_progress_log = {0}\u8FDB\u5EA6\u65E5\u5FD7
```

Comment:
Did you remove space between the parameter and character here on purpose?

Reply:
Yes, the space here is indeed removed intentionally, because Chinese characters are generally close to each other and do not use spaces to separate words like English.

Reply:
Thanks for the answer.